.vscode/settings.json

```diff
@@ -1,3 +1,5 @@
 {
-    "eslint.workingDirectories": ["services/app-web", "services/app-api"]
+    "eslint.workingDirectories": ["services/app-web", "services/app-api"],
+    "launchdarkly.project": "default",
```

Comment:
Is there a VSCode plugin related to LD?

Reply:
There is, it let's you do things like see flag toggle state and whatnot from VSCode: https://docs.launchdarkly.com/integrations/vscode#installing-the-vscode-extension